apps/coscrad-frontend/src/components/dynamicViews/dynamicResourceDetailView/__snapshots__/buildViewModelComponentFromCoscradDataDefinition.spec.tsx.snap

```diff
@@ -149,6 +149,11 @@ exports[`buildPresenterFromDataSchema should render successfully 1`] = `
         {"type":"one","strength":85}
       </div>
     </div>
+    <div>
```

Comment:
Excellent HTML!

Reply:
Back-end guy writing markup- alarms must be going off somewhere!